blackbelt/deployment.py

```diff
@@ -14,6 +14,11 @@ def deploy_staging():
 def deploy_production():
     post_message("Deploying to production", "#deploy-queue")
 
-    check_call(['grunt', 'deploy'])    
-    check_call(['grunt', 'deploy', '--app=apiary-staging-pre'])
-    check_call(['grunt', 'deploy', '--app=apiary-staging-qa'])
+    slug_creaction_rc = run_grunt_in_parallel((['grunt', 'create-slug'], ['grunt', 'create-slug', '--app=apiary-staging-pre'], ['grunt', 'create-slug', '--app=apiary-staging-qa']))
```

Comment:
I'd put the tuple into several lines for better readability:

```python
slug_creaction_rc = run_grunt_in_parallel((
    [...],
    [...],
    [...],
    [...],
)
```

Reply:
Also, from reading just this part of the code, I'm not quite sure what `slug_creaction_rc` means and what are its contents. I can see further below it's tested for not being zero, so I guess it's a count of something, but the name doesn't help me to understand why/what.

Reply:
Changed to longer, but more revealing name, thank you.